graphql-api/src/main/java/com/expediagroup/streamplatform/streamregistry/graphql/resolvers/DomainResolver.java

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
For some reason intellij is configured to perform this peculiar change. please avoid it on non-new-added files